awxkit/awxkit/api/client.py

```diff
@@ -57,7 +56,7 @@ def login(self, username=None, password=None, token=None, **kwargs):
             else:
                 self.session.auth = (username, password)
         elif token:
-            self.session.auth = Token_Auth(token, auth_type=kwargs.get('auth_type', 'Token'))
```

Comment:
This default is wrong, and doesn't even work anymore.

See: 28512e042b4dfe16ef75d39aea79f2d82459b3a9